tests/functional/settings_devices.js

```diff
@@ -145,6 +145,14 @@ define([
           );
         })
 
+        // current device has the text `current device`
+        .findByCssSelector('.device:nth-child(1) .device-name + .device-current')
```

Comment:
Can you replace lines 149-154 with a call to [testElementTextEquals](https://github.com/mozilla/fxa-content-server/blob/0ff2e355efb898c82822dc5bbfbdab9351f25b21/tests/functional/lib/helpers.js#L1034)